Makefile

```diff
@@ -50,6 +50,8 @@ else
 endif
 
 local-test: ## Alias for 'make test WORKER_TYPE=qemu'
+	ssh-keygen -q -t rsa -N '' -f ./core/id <<<y >/dev/null 2>&1
+	cp ./core/id.pub ./worker/id.pub
```

Comment:
This command is just an alias and may not always be used. Instead this logic should be moved to lines 46-47 above in the `ifeq ($(WORKER_TYPE),qemu)` block.

Reply:
@klutchell I found a way to make this step unnecessary now.